### ISSUE TITLE
color.data (categorical & primary/secondary/tertiary) need contrasting label pairings

### ISSUE THREAD

**Issue by mikeldn** [Greater-London-Authority/ldn-viz-tools#932]
**Is your feature request related to a problem? Please describe.**
When we set a shape (map / chart) to a categorical (or pri, sec, ter) color, and place a label on that shape, that label needs to contrast with the color. In light mode, a white label that works on blue, doesn't work on yellow. 

**Describe the solution you'd like**
Each theme/color token has a contrasting label color set, like:
- color.data.categorical.blue.label: #ffffff (white)
- color.data.categorical.yellow.label: #0E1316 (darkest grey)

**Describe alternatives you've considered**
We can of course calculate contrast on the fly for more fluid colors, but it would be good to have these pre-canned. 

**Additional context**
Maybe we could also have label colours for each colour palette step (color.palette.blue[500]) but probably unneeded.
